Source/WebCore/platform/graphics/avfoundation/AVTrackPrivateAVFObjCImpl.mm

```diff
@@ -295,10 +295,17 @@ - (id)optionID;
 
 AtomString AVTrackPrivateAVFObjCImpl::language() const
 {
-    if (m_assetTrack)
-        return AtomString { languageForAVAssetTrack(m_assetTrack.get()) };
-    if (m_mediaSelectionOption)
-        return AtomString { languageForAVMediaSelectionOption(m_mediaSelectionOption->avMediaSelectionOption()) };
+    if (m_assetTrack) {
+        auto language = languageForAVAssetTrack(m_assetTrack.get());
+        if (!language.isEmpty())
+            return AtomString { language };
+    }
+
+    if (m_mediaSelectionOption) {
+        auto language = languageForAVMediaSelectionOption(m_mediaSelectionOption->avMediaSelectionOption());
+        if (!language.isEmpty())
+            return AtomString { language };
+    }
 
     ASSERT_NOT_REACHED();
```

Comment:
This will assert if a track or media selection option has an empty or unknown language

Reply:
Good point! I'll remove the assert.